Assertion error while loading simple module with an indirection inside a class

**Describe the bug**
Assertion error while loading simple module with an import inside a class. 


**To Reproduce**
Steps to reproduce the behavior:
Try to parse the following module with `load_python_modules()`:

```python
    class session:
        from twisted.conch.interfaces import ISession
```

**Expected behavior**
A new module should be returned, with an Indirection inside a class.

**Actual bahaviour**

I get the following error

```python
self = <docspec_python.parser.Parser object at 0x106763f10>
parent = Module(name='test', members=[])
node = Node(classdef, [Leaf(1, 'class'), Leaf(1, 'session'), Leaf(11, ':'), Node(suite, [Leaf(4, '\n'), Leaf(5, '    '), Node...h'), Leaf(23, '.'), Leaf(1, 'interfaces')]), Leaf(1, 'import'), Leaf(1, 'ISession')]), Leaf(4, '\n')]), Leaf(6, '')])])
decorations = None

    def parse_classdef(self, parent, node, decorations):
      name = node.children[1].value
      bases = []
      metaclass = None
    
      # An arglist is available if there are at least two parameters.
      # Otherwise we have to deal with parsing a raw sequence of nodes.
      classargs = find(lambda x: x.type == syms.arglist, node.children)
      if classargs:
        metaclass, bases = self.parse_classdef_arglist(classargs)
      else:
        metaclass, bases = self.parse_classdef_rawargs(node)
    
      suite = find(lambda x: x.type == syms.suite, node.children)
      docstring = self.get_docstring_from_first_node(suite) if suite else None
      class_ = Class(
        name=name,
        location=self.location_from(node),
        docstring=docstring,
        metaclass=metaclass,
        bases=bases,
        decorations=decorations,
        members=[])
    
      for child in suite.children if suite else []:
        if isinstance(child, Node):
          member = self.parse_declaration(class_, child)
>         assert not isinstance(member, (list, Module)), member
E         AssertionError: [Indirection(name='ISession', target='twisted.conch.interfaces.ISession')]
```

It's really the indirection inside the class the issue, I've tried with the following, and it works as expected. 

```python
    from twisted.conch.interfaces import ISession
    class session:
        ISession=ISession
```

--- a/docspec-python/.changelog/_unreleased.yml
+++ b/docspec-python/.changelog/_unreleased.yml
@@ -1,0 +1,6 @@
+changes:
+- type: fix
+  component: general
+  description: support imports on the class-level
+  fixes:
+  - '#34'

--- a/docspec-python/src/docspec_python/parser.py
+++ b/docspec-python/src/docspec_python/parser.py
@@ -403,13 +403,16 @@ class Parser:
 
     for child in suite.children if suite else []:
       if isinstance(child, Node):
-        member = self.parse_declaration(class_, child)
-        assert not isinstance(member, (list, Module)), member
-        if metaclass is None and isinstance(member, Data) and \
-            member.name == '__metaclass__':
-          metaclass = member.value
-        elif member:
-          class_.members.append(member)
+        members = self.parse_declaration(class_, child) or []
+        if not isinstance(members, list):
+          members = [members]
+        for member in members:
+          assert not isinstance(member, Module)
+          if metaclass is None and isinstance(member, Data) and \
+              member.name == '__metaclass__':
+            metaclass = member.value
+          elif member:
+            class_.members.append(member)
 
     class_.metaclass = metaclass
     return class_

--- a/docspec-python/src/test/test_parser.py
+++ b/docspec-python/src/test/test_parser.py
@@ -352,6 +352,11 @@ def test_indirections():
   from ..core import Widget, View
   from .vendor import pkg_resources, six
   from ...api import *
+  def foo():
+    from sys import platform
+  class bar:
+    import os
+    from os.path import dirname
   """
 
   return [
@@ -371,4 +376,9 @@ def test_indirections():
     Indirection('pkg_resources', Location('test_indirections', 13), None, '.vendor.pkg_resources'),
     Indirection('six', Location('test_indirections', 13), None, '.vendor.six'),
     Indirection('*', Location('test_indirections', 14), None, '...api.*'),
+    Function('foo', Location('test_indirections', 15), None, None, [], None, []),
+    Class('bar', Location('test_indirections', 17), None, None, [], None, [
+      Indirection('os', Location('test_indirections', 18), None, 'os'),
+      Indirection('dirname', Location('test_indirections', 19), None, 'os.path.dirname'),
+    ])
   ]

